src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Impossible de charger le middleware.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **MiddlewareCantBeLoaded**: Le middleware a généré une erreur pendant qu'Astro essayait de le charger.
```

Comment:
```suggestion
> **MiddlewareCantBeLoaded**: The middleware threw an error while Astro was trying to loading it.
```
Correct! This part should be in English, exactly as the error message would be seen. Everything else *is* translated!

Reply:
I've juste made the change